packages/core/src/service/Config.ts

```diff
@@ -26,6 +26,29 @@ export interface Config {
 	snippet: SnippetsConfig
 }
 
+export interface CustomResourceConfig {
+	/**
+	 * The entry you have dispatched from mcdoc. (eg. `picoblaze:game_event`)
+	 */
+	category: string
+	/**
+	 * The file extension name of the dispatched resource. Only `.json` is supported for now.
+	 */
+	extname: '.json'
+	/**
+	 * The pack type of the dispatched resource. Only `datapack` is supported for now.
+	 */
+	pack: 'datapack'
+	/**
+	 * The version range of the dispatched resource.
+	 */
+	since?: `1.${number}`
```

Comment:
If we get 10 patch versions, how do we differentiate 1.x.1 and 1.x.10 with this format?

Reply:
Well this is leftover from before my PR, I think the idea here was to not track minor versions in this file at all? I'd be up for changing this though

Reply:
Looks like there's usages of minor versions via `ReleaseVersion` already, so I'll resolve this. Added some resources that include the minor version too.